src/__tests__/FileUpload.test.tsx

```
@@ -1,851 +1,371 @@
-  });
 
-  describe('Edge Cases and Error Conditions', () => {
-    test('handles empty file selection gracefully', async () => {
-      const mockOnFileSelect = vi.fn();
-      
+    it('handles very long file names', async () => {
+      const user = userEvent.setup();
       render(<FileUpload onFileSelect={mockOnFileSelect} />);
       
-      const fileInput = screen.getByRole('button').querySelector('input[type="file"]') as HTMLInputElement;
+      const longFileName = 'a'.repeat(255) + '.txt';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)